website/index.html

```diff
@@ -224,6 +225,7 @@
       <div class="col-md-7">
         <div class="row header-graphic">
           <img class="img-fluid mountain">
+          <img src = "assets/images/flat-mountain-graphic.svg">
```

Comment:
add this src to the above line instead. This will generate two images, one which will always be visible regardless.